tests/seq_io/test_genbank.gbk

```diff
@@ -0,0 +1,90 @@
+LOCUS       SCU49845     5028 bp    DNA             PLN       21-JUN-1999
```

Comment:
To test the case of unavailable tag on the destination. Genbank format is in SeqInFormat  but not in SeqOutFormat. I wanted the SeqFileIn format to be detected automatically before trying to transfer the tag to the target format.